src/main/proto/commit.proto

```diff
@@ -24,5 +24,7 @@ message Commit {
     string author_email = 4;
     string date = 5;
     bool qommit = 6;  // Quality commit.
-    repeated CommitStats commit_stats = 7;  // Stats on a commit.
+    int32 num_lines_add = 7;  // Number of lines for whole commit.
```

Comment:
These are new and they are not obviously useful. Presumably you can compute them by aggregating Stats. Usually, when you respond to comments in code review, you should refrain from adding unexpected non-obvious changes (i.e. changes that don't pertain to review comments), because this will inevitably delay the process. If you HAVE to add those, please explain why in your commit comment.

I am going to approve this now, but please remove these new fields, and include them in a future commit if they are actually needed.

Don't forget to squash the dev branch before pushing. Also - should we merge or rebase? For now we should probably do rebase because it ends up in a cleaner master.